src/components/AppMap.ts

```diff
@@ -834,9 +836,50 @@ export default class AppMap extends mixins(MixinUtil) {
       marker.data.getMarker().addTo(this.map.m);
     }
 
+    this.updateTooltip();
     this.searching = false;
   }
 
+  yTooltipOnEach(marker: any) {
+    let m: any = marker.getMarker();
+    if (!('_tooltip' in marker.obj)) {
+      // @ts-ignore
+      let tt = m.getTooltip();
+      marker.obj._tooltip = tt.getContent();
+      marker.obj._tooltip_options = tt.options;
+    }
+    m.unbindTooltip();
+    m.bindTooltip(`${marker.obj.pos[1]}`, { permanent: true });
+    m.openTooltip();
+  }
+
+  yTooltipOffEach(marker: any) {
+    let m: any = marker.getMarker();
+    m.getTooltip().options.permanent = false;
+    m.unbindTooltip();
+    m.bindTooltip(marker.obj._tooltip, marker.obj._tooltip_options);
+    m.closeTooltip();
+  }
+
+  yTooltipSwitch(on: boolean) {
+    let func = (on) ? this.yTooltipOnEach : this.yTooltipOffEach;
+    this.searchResultMarkers.map(m => m.data).forEach(func);
+    this.searchGroups.forEach(group => {
+      group.getMarkers().forEach(func);
+    });
+  }
+
+  updateTooltip() {
+    if (this.staticTooltip) {
+      this.yTooltipSwitch(this.staticTooltip);
+    }
+  }
+
+  toggleY() {
+    this.staticTooltip = !this.staticTooltip;
+    this.yTooltipSwitch(this.staticTooltip);
```

Comment:
If you remove the check in updateTooltips(), this can become a call to updateTooltips().